core/domain/user_services.py

```diff
@@ -863,7 +996,8 @@ def update_multiple_users_data(modifiable_user_data_list):
             not found.
     """
     user_ids = [user.user_id for user in modifiable_user_data_list]
-    user_settings_list = get_users_settings(user_ids)
+    user_settings_list = get_users_settings(user_ids, strict=False)
+    strict_user_settings_list = []
```

Comment:
This seems off -- again, typing should not make it to the "variable name" or "code impl" level.

Reply:
Variable names are changed, Acc. to the new discussion!